plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -219,46 +219,47 @@ SecRule REQUEST_FILENAME "@contains /apps/recommendations/settings/enabled" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
-# Text app autosave sync feature doesn't works 
+# Text app autosave sync feature doesn't works
 SecRule REQUEST_URI "@contains /apps/text/session/sync" \
     "id:9508126,\
     phase:1,\
     pass,\
     t:none,\
     nolog,\
-    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
-    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:autosaveContent"
+    ctl:ruleRemoveTargetById=941100;ARGS:json.documentState,\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:autosaveContent,\
```

Comment:
Please add a comment to the rule to document which argument names concern which versions of Nextcloud.

Reply:
fixed